language-python.md

```diff
@@ -400,3 +400,19 @@ If a function argument's name clashes with a reserved keyword, it is generally b
 #### Constants
 
 Constants are usually defined on a module level and written in all capital letters with underscores separating words. Examples include `MAX_OVERFLOW` and `TOTAL` .
+
+#### String format
+
+We recommend to use `format()` instead of `%`.
+
+Recommend:
+```python
+'{} {}'.format('one', 'two')
```

Comment:
I’m not familiar with Python, Should there be something in `{}` or are these just generic placeholders?

I would expect `{one}` and `{two}`. :)